tools/structure/src/main.py

```diff
@@ -79,8 +80,11 @@ def run(
         # TODO: Need to split extraction and indexing
         # to avoid unwanted indexing
         self.stream_log("Indexing document")
+        usage_kwargs: dict[Any, Any] = dict()
+        run_id = CommonUtils.generate_uuid()
+        usage_kwargs["run_id"] = run_id
```

Comment:
@gaya3-zipstack can you add `workflow_id` and `execution_id` as well? Until we have the new metrics changes come in - we'll atleast populate the usage table correctly.
This can be obtained from [these attributes in BaseTool](https://github.com/Zipstack/unstract-sdk/blob/29a22c682c0cd5ae090ff375db217bfcd2fa2e9f/src/unstract/sdk/tool/base.py#L37-L38)

Reply:
Ideally, I think run_id also does not make sense in the tools world. We need to add workflow and execution id and make sure it works properly end to end. Will take this up in a separate PR as discussed in the call @chandrasekharan-zipstack